src/Drivers/CardsDriver.cpp

```diff
@@ -0,0 +1,110 @@
+//
+// Created by claudia on 9/12/2019.
+//
+
+#include <iostream>
+#include "../../include/Cards.h"
+
+int main() {
+
+    int infantryCount = 0;
+    int artilleryCount = 0;
+    int cavalryCount = 0;
+
+    Deck deck = Deck(9);
+    deck.createDeck();
+
+    int size = deck.getNumberOfCards();
+
+    std::cout << "\n--------------------\n Creation of a Deck \n--------------------" << std::endl;
+
+    for (int i = 0; i < size; i++) {
+
+        if (deck.getDeck()->at(i) == 0) {
+            std::cout << "Card: [INFANTRY]" << std::endl;
+            infantryCount++;
+        }if (deck.getDeck()->at(i) == 1) {
+            std::cout << "Card: [ARTILLERY]" << std::endl;
+            artilleryCount++;
+        }if (deck.getDeck()->at(i) == 2) {
+            std::cout << "Card: [CAVALRY]" << std::endl;
+            cavalryCount++;
+        }
+    }
+
+    std::cout << "\n\n------------------\n Deck Composition \n------------------" << std::endl;
+
+    printf ("Number of [INFANTRY] cards: [%d]", infantryCount);
```

Comment:
Why the use of `printf` here over `std::cout`? Paquet said we should be using `std::cout` since `printf` isn't very "c++-y".

Reply:
Is there a way to substiture variables or should we do string + variable + string? I've always been told that doing that was ugly so I looked for something which allowed for variable substitution

Reply:
You can also do: 
```suggestion
    std::cout "Number of [INFANTRY] cards: " << infantryCount;
```
It's not pretty, but I think this would be the "proper" C++ way of doing it. Can also pipe an extra `"\n"` at the end if you want a newline.